src/openrct2-ui/audio/AudioMixer.cpp

```diff
@@ -481,7 +479,7 @@ namespace OpenRCT2::Audio
             for (int32_t i = 0; i < length; i++)
             {
                 float t = (float)i / length;
-                data[i] = (uint8_t)(data[i] * ((1 - t) * startvolume_f + t * endvolume_f));
+                data[i] = static_cast<float>(data[i]) * ((1 - t) * startvolume_f + t * endvolume_f);
```

Comment:
Why not 1.0?

Reply:
Consistency with the other casts, I guess, but multiplication with a float works too. Specific preference?

Reply:
I would prefer that the floats are used with floats and I think that your cast is the wrong way, data is uint8_t and not float

Reply:
Ouch, that is a plain error. I will check this in context and fix it.